src/main/java/de/zalando/ep/zalenium/proxy/DockerSeleniumRemoteProxy.java

```diff
@@ -41,8 +41,12 @@
 import java.io.OutputStream;
 import java.util.Map;
 import java.util.Optional;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.logging.Level;
+import java.util.logging.Logger;
```

Comment:
It think we want to use slf4j. @dennisgranath recently made changes to start using it instead of java.util.

Reply:
Yeah that's true

Reply:
Will change.